### ISSUE TITLE
AGPL Licence and questions about commercial use

### ISSUE THREAD

**Issue by sebarthel89** [renovatebot/renovate#5752]
Hey there,

I really like your project and how to approach all the complicated issues around dependency management.

I have a question about commercial use. Right now, you use an AGPL Licence that is a copyleft license. Are companies allowed to use your docker container on private servers on their private code? I met different people with different opinions. It would be great if you clarify the restrictions for commercial private hosted use.

Thanks in advance!

**Comment by rarkins** [comment]
As a disclaimer: the license itself (i.e. AGPL) is ultimately what applies, and nothing I say here is legal advice or overrides the license text.

Personal opinion on AGPL generally and not Renovate specifically follows:

The AGPL license itself definitely does not exclude commercial or private use. Many references for this including https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)

As with any xGPL, any "viral" conditions you may have doubts about only matter if you are tightly integrating the GPL'd code. Running a pre-built docker image is pretty close to the furthest you can get from such a condition, and especially as you would not be running a service to anybody publicly anyway.

In short: unless you are customizing/modifying the AGPL'd source code itself, it shouldn't make any difference to you.

**Comment by sebarthel89** [comment]
Thanks for the fast response, Keep up the good work 👍
